togetherly/lib/services/reward_redemption_service.dart

```diff
@@ -0,0 +1,58 @@
+import 'package:supabase_flutter/supabase_flutter.dart';
+import 'package:togetherly/models/reward_redemption.dart';
+
+class RewardRedemptionService {
+  static const String _rewardRedemptionTable = "redeemed_reward";
+  static const String _familyRewardRedemptionView = "family_redeemed_reward";
+
+  Future<List<RewardRedemption>> getRewardRedemptionsByFamily(
+      int familyId) async {
+    var result = await Supabase.instance.client
+        .from(_familyRewardRedemptionView)
+        .select('id, person_id, reward_id, quantity, timestamp')
```

Comment:
The `family_redeemed_reward` view is currently missing an `id` column, so this method isn't working. Could you add that column to the database, @ryan61?

Reply:
I added it.